Task_HelloWorld/index.html

```diff
@@ -7,5 +7,11 @@
 
```

Comment:
The opening <h> tag should have a closing </h> tag to properly enclose the heading.